client/src/components/location.jsx

```diff
@@ -1,46 +1,136 @@
+  flex-direction: row;
+  /* overflow: visible; */
+  height: 500px;
+`
+const FactMill = Styled.button`
+  transition: 0.3s;
+  border-radius: 20px;
   border: none;
   background-color: transparent;
+  font-family: 'Nunito Sans', sans-serif;
+  margin: 15px;
+  padding: 5px 20px;
+`
+const LearnMore = Styled(FactMill)`
+  background-color: #fff;
+  font-family: 'Nunito Sans', sans-serif;
+`
+const Company = Styled.div`
+  transition: 0.3s;
+  grid-area: company; 
+  background-color: #f9f9f9;
+  margin-left: 20px;
+`
+const Country = Styled(Company)`
+  grid-area: country;
+  opacity: 0;
+`
+const CardContainer = Styled.div`
+  background-color: #f9f9f9;
+  font-family: 'Nunito Sans', sans-serif;
+  cursor: pointer;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 0.2fr 1.5fr 1.5fr 0.7fr;
+  grid-template-rows: 0.3fr 0.3fr 1.7fr 1.7fr;
+  gap: 0px 0px;
+  grid-template-areas:
+    "img img img img"
+    "company company company company"
+    "country country country country"
+    "footer footer footer footer";
+  transition: 0.3s;
+  width: 350px;
+  border-radius: 5%; 
+  margin: 10px;
+  &:hover {
+    transform: scale(1.015);
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  }
+  &&:hover ${LearnMore} {
+    background-color: #000;
+    color: #fff;
+  }
+  &&:hover ${Country} {
+    opacity: 1;
+  }
 `
-const RightButton = styled.button`
+const CardInfo = Styled.div`
+  grid-area: footer;
+  align-self: start;
   display: flex;
-  border-radius: 10px;
-  border: none;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: none;
+  transition: all 0.2s ease-in-out; 
+  border-radius: 0 0 5% 5%;
+`
+const Img = Styled.img`
+  grid-area: img;
+  height: 85%; 
+  width: 100%; 
+  border-radius: 5% 5% 0px 0px;
 `
 
 class Location extends React.Component {
   constructor(props) {
     super(props);
-
   }
 
   render() {
     let locations = this.props.locations;
-    let pics = locations.map(location => {
-      const { loc_pic, loc_company, loc_name, id } = location;
-      return (
-        <div className='card' key={id}>
-          <img src={loc_pic} />
-          <div>{loc_name}</div>
-          <div>{loc_company}., Ltd</div>
-          <div className='two-buttons'>
-            <LeftButton>Learn More</LeftButton>
-            <RightButton>Factory</RightButton>
-          </div>
-        </div>
-      )
-    })
-    console.log(locations);
+    if(!locations.length) {
+      return <Carousel>
+        <CardContainer key='no'>
+          <Img src="https://pixabay.com/get/ge9ac0a1608e17c9cdcec58ce2b3d814b6d579bdca6d849b5fe516e52d21a367338f7db585786311f105f9e6de559fb9c_640.jpg"></Img>
+          <Company>Tater Tots Co., Ltd.</Company>
+          <Country>Hack Reactor</Country>
+          <CardInfo>
+            <FactMill>Factory</FactMill>
+            <LearnMore>Learn More</LearnMore>
+          </CardInfo>
+        </CardContainer>
+      </Carousel>
+    }
+    let pics = locations.length > 2 
```

Comment:
i think this should be DRYer by just mapping once here regardless of length, and slicing off three when you need them

Reply:
@danielharmon Ha!! Didn't think of it. Will make a change.